src/parser/ast.rs

```diff
@@ -958,7 +958,7 @@ impl fmt::Display for QueryTarget {
         match self {
             Internal(v) => v.fmt(f),
             External(prefix) => match prefix {
-                PathPrefix::Event => write!(f, "."),
+                PathPrefix::Event => write!(f, ""),
```

Comment:
Should probably just remove this `write!` all together? If I understand correctly, and based on my testing `QueryTarget` always appears with a `.` after it even if `OwnedValuePath` is empty. So this is safe.